modules/apps/portal/portal-util-test/src/testIntegration/java/com/liferay/portal/util/test/PortalImplAlternateURLTest.java

```diff
@@ -114,6 +115,55 @@ public static void tearDownClass() {
 			GetterUtil.getString(_defaultPrependStyle));
 	}
 
+	@Test
+	public void testAlternateURLDoesNotHaveLocaleInHomeLayout()
+		throws Exception {
+
+		_group = GroupTestUtil.addGroup();
+
+		Collection<Locale> availableLocales = Arrays.asList(
+			LocaleUtil.BRAZIL, LocaleUtil.US);
+		Locale defaultLocale = LocaleUtil.US;
+
+		_group = GroupTestUtil.updateDisplaySettings(
+			_group.getGroupId(), availableLocales, defaultLocale);
+
+		Layout layout = LayoutTestUtil.addTypePortletLayout(
+			_group.getGroupId(), false,
+			HashMapBuilder.put(
+				LocaleUtil.BRAZIL, "Inicio"
+			).put(
+				LocaleUtil.US, "Home"
+			).build(),
+			HashMapBuilder.put(
+				LocaleUtil.BRAZIL,
+				StringPool.SLASH.concat(
+					FriendlyURLNormalizerUtil.normalize(
+						RandomTestUtil.randomString(
+							LayoutFriendlyURLRandomizerBumper.INSTANCE)))
+			).put(
+				LocaleUtil.US,
+				StringPool.SLASH.concat(
+					FriendlyURLNormalizerUtil.normalize(
+						RandomTestUtil.randomString(
+							LayoutFriendlyURLRandomizerBumper.INSTANCE)))
+			).build());
+
+		ThemeDisplay themeDisplay = _getThemeDisplay(_group, layout);
+
+		_testAlternateURLWithHomeLayout(
+			availableLocales, defaultLocale, themeDisplay, 0);
+
+		_testAlternateURLWithHomeLayout(
+			availableLocales, defaultLocale, themeDisplay, 1);
+
+		_testAlternateURLWithHomeLayout(
+			availableLocales, defaultLocale, themeDisplay, 2);
+
```

Comment:
The new lines between the calls to `_testAlternateURLWithHomeLayout` should be removed to match the formatting of the rest of the class.